website/databend/docs/sqlstatement/hash-functions/md5has.md

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Great work!
The document file name is md5hash not md5hash?

Reply:
Sorry, typos. Fixed.